app/api/datastore/entities/search_result.rb

```diff
@@ -2,6 +2,23 @@ module Datastore
   module Entities
     class SearchResult < Grape::Entity
       expose :id, as: :resource_id
+      expose :submitted_at
+      expose :applicant_name
+      expose :reference
+
+      private
+
+      def applicant_name
+        "#{applicant&.dig('first_name')} #{applicant&.dig('last_name')}"
```

Comment:
I _think_ .dig has the null safety baked into it

Reply:
But not if the hash itself is null.

Reply:
Aha, ok 👌